backstory/src/main/java/com/google/sps/story/PromptManagerAPIsClient.java

```diff
@@ -25,34 +25,30 @@
  * Facilitates API calls through a single object.
  */
 public class PromptManagerAPIsClient {
-  /** Client object for word classification */
-  private NLServiceClient wordClassifier;
   /** Client object for word fetching */
   private DatamuseRequestClient wordFetcher;
 
   /**
-   * Initialize API objects.
-   *
-   * @throws IOException Exception for network problem.
+   * Initialize Datamuse API object.
    */
   public PromptManagerAPIsClient() throws IOException {
```

Comment:
If the throws tag is still present, throws should still be in the javadoc?

Reply:
The throws tag should not still be present. I'll remove it. Thanks!

Reply:
Done